DNN Platform/Modules/BulkInstall/BulkInstall.Web/src/components/tabs/bulk-install-install/bulk-install-install.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'bulk-install-install',
+  styleUrl: 'bulk-install-install.scss',
+  shadow: true,
+})
+export class BulkInstallInstall {
+
+  render() {
+    return (
+      <Host>
+        <div class="row">
+          <div class="col">
+            <div class="panel">
+              <div class="panel-heading">
+                <h3 class="panel-title">Upload Install Package(s)</h3>
+              </div>
+              <div class="panel-body">
+                <dnn-dropzone allowed-extensions="zip"></dnn-dropzone>
+                <div class="form-group">
+                  <dnn-button>Install</dnn-button>
```

Comment:
I think I already created a localization key for this that we can use now

Reply:
Same for reset I think

Reply:
I see a key for `Install`, but not `Reset` yet.  Is this something we can handle in a future PR?

Reply:
Yes we can